infra/cifuzz/build_fuzzers_test.py

```diff
@@ -97,18 +97,20 @@ class InternalGithubBuildTest(unittest.TestCase):
   COMMIT_SHA = 'fake'
   PR_REF = 'fake'
 
-  def _create_builder(self, tmp_dir):
+  def _create_builder(self, tmp_dir, oss_fuzz_project_name=PROJECT_NAME):
```

Comment:
Can you delete the PROJECT_NAME class attr on line 94 please?

Reply:
done!